modules/search/Search.py

```diff
@@ -0,0 +1,80 @@
+class Search:
+    """Performs search algorithm"""
+
+    def __init__(self):
+        pass
+
+    def perform_search(self, tentGPS, planeGPS, angle):
+
+        """
+        Search program to determine the minimum rotation angle between heading and tent
+
+        Parameters
+        ----------
+        tentGPS : obj
+          dictionary containing lat and long coordinates of the tent
+        planeGPS : obj
+          dictionary containing lat and long coordinate of the plane
+        angle : float
+          bearings notation in degrees
+
+        Returns
+        -------
+        dict<"angle" : float, "counterclockwise" : bool>
+            Returns a dict with angle of rotation and whether rotation is clockwise or counterclockwise
+
+                "angle" : float
+                    Angle of rotation specified
+
+                "counterclockwise" : bool
+                    True if rotation should be counterclockwise
+                    False if rotation should be clockwise
+
+        """
+        import math
+
+        
+        '''
+            Calculating bearings between plane and tent
+        '''
+        planeLat = planeGPS["lat"]*math.pi/180
+        planeLon = planeGPS["lon"]
+
+        tentLat = tentGPS["lat"]*math.pi/180
+        tentLon = tentGPS["lon"]
+
+        diffLon = (tentLon - planeLon)*math.pi/180
```

Comment:
This makes sense except for the *math.pi/180, same as above, I'm assuming this is a radian conversion?

Reply:
As mentioned in the previous comment, we need our latitude and longitude values to be in radians to work with the math library functions.

The values for the latitude of the tent and plane are used individually throughout the algorithm (which is why each one was converted into radians). 

However for the longitude values we only really care about the difference in longitude between that of the plane and the tent, thus we first calculated the difference and converted that into radians.